src/browser.js

```diff
@@ -53,9 +55,9 @@ export default function cssHasPseudo(document) {
 							nodes.push(element);
 
 							// set an attribute with an irregular attribute name
-							attributeElement.innerHTML = '<x ' + item.attributeName + '>';
+							attributeElement.outerHTML = '<x ' + item.attributeName + '>';
```

Comment:
This is a really clever idea. Unfortunately, `outerHTML` cannot be set on an element without a parent, so I’m afraid this might not work.